aspnet/mvc/overview.rst

```diff
@@ -1,8 +1,313 @@
+
+.. note:: It's not uncommon for controllers to become bloated with too many responsibilities. Try to follow the `Single Responsibility Principle <http://deviq.com/single-responsibility-principle/>`_ and push business logic out of the controller and into the domain model whenever possible.
+
+.. tip:: If you find that your controller actions frequently perform the same kinds of actions, you can follow the `Don't Repeat Yourself principle <http://deviq.com/don-t-repeat-yourself/>`_ by moving these common actions into `filters`_.
+
+View Responsibilities
+^^^^^^^^^^^^^^^^^^^^^
+
+Views are responsible for presenting content through the user interface. There should be minimal logic within views, and any logic in them should relate to presenting content. If you find the need to perform a great deal of logic in view files in order to display data from a complex model, consider using a ViewModel instead that is designed to suit the needs of the View.
+
+What is ASP.NET Core MVC
+------------------------
+
+The ASP.NET Core MVC framework is a lightweight, open source, highly testable presentation framework optimized for use with ASP.NET Core. It is available in the "Microsoft.AspNetCore.Mvc" package, and is used by Visual Studio's "Web API" and "Web Application" ASP.NET Core Templates.
+
+ASP.NET Core MVC gives you a powerful, patterns-based way to build dynamic websites that enables a clean separation of concerns and gives you full control over markup for enjoyable, agile development. ASP.NET Core MVC includes many features that enable fast, TDD-friendly development for creating sophisticated applications that use the latest web standards.
+
+ASP.NET Core MVC in ASP.NET Core includes support for building web pages and HTTP services in a single aligned framework that can be hosted in IIS or self-hosted in your own process.
+
+Features
+--------
+
+ASP.NET Core MVC includes the following features:
+  - `Routing`_
+  - `Model binding`_
+  - `Model validation`_
+  - `Dependency injection`_
+  - `Filters`_
+  - `Areas`_
+  - `Web APIs`_
+  - `Testability`_
+  - `Razor view engine`_
+  - `Strongly typed views`_
+  - `Mobile support`_
+
+Routing
+^^^^^^^
+
+ASP.NET Core MVC is built on top of :doc:`ASP.NET Core's routing </fundamentals/routing>`, a powerful URL-mapping component that lets you build applications that have comprehensible and searchable URLs. This enables you to define your application's URL naming patterns that work well for search engine optimization (SEO) and for link generation, without regard for how the files on your web server are organized. You can define your routes using a convenient route template syntax that supports route value constraints, defaults and optional values.
```

Comment:
I think it's important to note that MVC is built on top of routing (and also not to make it appear that routing is an MVC-only feature, since it's not).